tests/test_czml.py

```diff
@@ -2,18 +2,73 @@
 import sys
 
 from astropy import units as u  # noqa: E402
-
 from poliastro.bodies import Mars
 from poliastro.examples import iss, molniya  # noqa: E402
 
 try:
     from poliastro.czml.extract_czml import CZMLExtractor  # noqa: E402
-
+    from czml3.core import Document
     import czml3  # noqa: F401
 except ImportError:
     pass
 
 
+@pytest.mark.skipif("czml3" not in sys.modules, reason="requires czml3")
+def test_czml_get_document():
+    start_epoch = iss.epoch
+    end_epoch = iss.epoch + molniya.period
+
+    sample_points = 10
+
+    pr_map_url = (
+        "https://upload.wikimedia.org/wikipedia/commons/c/c4/Earthmap1000x500compac.jpg"
+    )
+    scene = False
+    # expected_packet = """[
```

Comment:
What's this long comment?

Reply:
Sorry I think I left it there while debugging.